translation/es-ES.toml

```diff
@@ -1,27 +1,35 @@
 name = "Español" # Name of language
 direction = "ltr" # Change to "rtl" for right to left languages
-version = "21"
+version = "23"
 maintainer = "xa31"
 
 [header]
 home = "Inicio"
 play = "Jugar"
 news = "Noticias"
 login = "Iniciar Sesión"
+profile = "Perfíl"
 createaccount = "Crear Cuenta"
+logout = "Cerrar sesión"
+
+[header.settings]
+language = "Idioma"
+board = "Tablero" # Board color/theme
+legalmoves = "Jugadas Legales" # Legal moves shape
+legalmoves-squares = "Casillas"
+legalmoves-dots = "Puntos" # Dots and 4 corner triangles
+perspective = "Perspectiva" # Perspective-mode
+perspective-mouse-sensitivity = "Sensibilidad del ratón"
+perspective-fov = "Campo de visión (FOV)"
+ping = ["Latencia", "ms"] # A number is inserted between these 2 strings.
+reset-to-default = "Reiniciar opciones por defecto"
```

Comment:
I don't think the "Reset to default" button will have enough space for this in the dropdown, or the button will be MASSIVE. Can this be simplified?
![Screenshot 2024-12-06 at 1 18 54 PM](https://github.com/user-attachments/assets/be424393-0f51-458f-b453-98d3db442713)

Reply:
I'll look at it